proto/forwarding/forwarding_common.proto

```diff
@@ -178,6 +180,9 @@ enum PacketFieldNum {
   PACKET_FIELD_NUM_HOST_PORT_ID = 63; // Host port id (metadata).
   PACKET_FIELD_NUM_L2MC_GROUP_ID = 64; // L2MC Group id (metadata).
   PACKET_FIELD_NUM_POLICER_ID = 65; // Policer id (metadata).
+  PACKET_FIELD_NUM_MPLS_LABEL = 66; // MPLS Label
+  PACKET_FIELD_NUM_MPLS_TC = 67; // MPLS Traffic Class
```

Comment:
Do we also want to add the bottom of stack field?

Reply:
maybe later, we currently set it automatically